app/src/main/java/project/pb/fragments/OpenDayInformation.java

```diff
@@ -29,6 +30,7 @@
     private TextView generalInfo;
     private ImageButton shareButton;
     private OpenDagData key;
+    private LinearLayout newopendaypage;
```

Comment:
I would have used constraintlayout, but this is even better for the functionality, goodthinking hamza!